style.css

```diff
@@ -121,3 +164,214 @@ body {
   margin-top: 10px;
   flex-grow: 1;
 }
+
+.grid-works {
+  display: grid;
+  grid-template-columns: 320px;
+  grid-template-rows: 100px 342px 342px 342px 342px 342px 342px;
+  margin-top: 5px;
+  margin-bottom: 5px;
+}
+
+.div-title {
+  text-align: center;
+  line-height: 100px;
+}
+
+.container-work {
+  display: flex;
+  flex-direction: column;
+  width: 320px;
+  height: 342px;
```

Comment:
Let's make the width a bit flexible by using [`max-width`](https://www.w3schools.com/cssref/pr_dim_max-width.asp) or [`min-width`](https://www.w3schools.com/cssref/pr_dim_min-width.asp), which ever you prefer.